{"run_number": "1713684804749879", "step_name": "end", "task_id": "7", "attempt_id": 0, "name": "_exception", "content_type": "gzip+pickle-v2", "type": "metaflow.artifact", "sha": "f3627f46179fdd95bf0e83101840fd1d71b60e40", "ds_type": "local", "location": ":root:/home/shashwat/Desktop/Newest Microservice/.metaflow", "flow_id": "DataFlow", "user_name": "shashwat", "tags": [], "system_tags": ["python_version:3.11.7", "user:shashwat", "runtime:dev", "metaflow_version:2.11.10"], "ts_epoch": 1713684816600}